edge-apps/rss-reader/screenly.yml

```diff
@@ -0,0 +1,46 @@
+app_id: 01H5X26SAC194MEJDFSMV7BFTD
+root_asset_id: 01H5X2X3VHA3AJD84TZSHD4ZV9
+user_version: v1.0.0
+revision: 1
+description: Screenly RSS Reader App
+icon: ''
+author: Screenly, Inc.
+homepage_url: ''
+settings:
+  disable_analytics:
+    type: text
+    default: "false"
+    optional: true
+    title: Disable Analytics
+    help_text: Whether to disable Sentry and Google Analytics integrations
+  ga_api_key:
+    type: text
+    default: ''
+    optional: true
+    title: Google Analytics API key
+    help_text: |
+      Specify a Google Analytics API key that will be used for the Google Tag Manager API.
+  sentry_id:
+    type: text
+    default: ''
+    optional: true
+    title: Sentry ID
+    help_text: Specify your Sentry ID
+  rss_title:
+    type: text
+    default: ''
+    optional: false
+    title: RSS Feeds Title
+    help_text: The title of the websote where the feeds were derived
+  rss_url:
+    type: text
+    default: ''
+    optional: false
+    title: RSS Feed URL
+    help_text: Specify the URL to where to get the data from.
+  limit:
+    type: text
+    default: '1'
+    optional: true
```

Comment:
Not optional as it can't operate without this.

Reply:
Acknowledged, thanks.